content/en/05-reference/01-admin/12-verifying-images.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Verifying Images"
+metaTitle: "Reference | Admin | Verifying Images"
+metaDescription: "Verifying container images"
+order: 12 
+---
+
+Pixie’s container images are signed with [cosign](https://github.com/sigstore/cosign). Each image is signed with Pixie’s private key and can be verified using Pixie’s public key. This can help you ensure the images you are running and deploying to your cluster are trustworthy.  
+
+## Verifying a Container 
+
+1. Download cosign by following the instructions [here](https://docs.sigstore.dev/cosign/installation/).
+
+2. Download Pixie’s public key:
+
+```bash
+wget https://raw.githubusercontent.com/pixie-io/px.dev/main/static/cosign.pub
```

Comment:
`wget https://px.dev/cosign.pub` instead
 ... actually you don't even need to wget, I believe you can just do `--key https://px.dev/cosign.pub` for the `cosign verify` command